src/encoded/static/components/body_map.js

```diff
@@ -0,0 +1,766 @@
+                            if (bodyClass.indexOf('cls') === -1) {
+                                multipleAssociations.push(bodyClass);
+                                if (active) {
+                                    document.getElementById(bodyClass).classList.add('active');
+                                } else {
+                                    document.getElementById(bodyClass).classList.remove('active');
+                                }
+                            }
+                        });
+                    }
+                }
+            });
+
+            // Set state to be new organ and any other matches
+            // We need to check if there is a new organ found because there are a very very few shapes that do not have corresponding organ_slims and in that case we do not want to add an undefined value to the state / try to navigate to undefined url
+            // For example, the uterine walls and vas deferens shapes do not correspond to any organ slim
+            if (newOrgan) {
+                const previousState = this.state.selectedOrgan;
+                // If there is no "currentOrgan" in state, we add the new organ and its associated terms
+                if (this.state.currentOrgan === []) {
+                    this.setState({
+                        selectedOrgan: [newOrgan, ...multipleAssociations],
+                    });
+                // If the state already includes the newly selected organ, we remove it and its associated terms
+                } else if (this.state.selectedOrgan.includes(newOrgan)) {
+                    if (typeof this.state.selectedOrgan !== 'string') {
+                        this.setState(prevState => ({
+                            selectedOrgan: prevState.selectedOrgan.filter(organ => organ !== newOrgan && !(multipleAssociations.includes(organ))) || [],
+                        }));
+                    } else {
+                        this.setState({ selectedOrgan: [] });
+                    }
+                // If there already is a "currentOrgan" but it does not include the clicked-on organ, add the new organ and its assocations
+                } else {
+                    let newState;
+                    if (typeof this.state.selectedOrgan !== 'string') {
+                        newState = [...this.state.selectedOrgan, ...multipleAssociations, newOrgan];
+                    } else {
+                        newState = [this.state.selectedOrgan, ...multipleAssociations, newOrgan];
+                    }
+                    this.setState({
+                        selectedOrgan: [...new Set(newState)],
+                    });
+                }
+
+                // Append or remove organ(s) to / from the search query
+                const parsedUrl = url.parse(this.props.context['@id']);
+                const query = new QueryString(parsedUrl.query);
+                if (active) {
+                    query.addKeyValue(organField, newOrgan, '');
```

Comment:
For a tiny bit of simplicity, you can just leave off the third parameter to `addKeyValue`. It’s just looking for something truthy.